incubator/jenkins/values.yaml

```diff
@@ -0,0 +1,31 @@
+# Default values for jenkins.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+Master:
+  Name: jenkins-master
+  Image: "gcr.io/kubernetes-charts-ci/jenkins-master-k8s"
+  ImageTag: "v6"
+  ImagePullPolicy: "Always"
+  Component: "jenkins-master"
+  AdminUser: admin
+# AdminPassword: <defaults to random>
+  Cpu: "200m"
```

Comment:
I think we can set via `JAVA_OPTS` ENV var with this docker.  Can you double check that in the jenkins.sh they have that.   When can then set:

```shell
JAVA_OPTS="-Xms512m -Xmx512m" 
```
You will need about 256m head room on top of that.  But it is all a guess.  Depending on what you are doing with Jenkins you could need as much as 2.5G with 3G total.